programs/locker/src/instructions/create_vesting_escrow.rs

```diff
@@ -45,6 +47,39 @@ impl CreateVestingEscrowParameters {
 
         Ok(())
     }
+
+    pub fn init_escrow(
+        &self,
+        vesting_escrow: &AccountLoader<VestingEscrow>,
+        recipient: Pubkey,
+        mint: Pubkey,
+        sender: Pubkey,
+        base: Pubkey,
+        escrow_bump: u8,
+        token_program_flag: TokenProgramFLag,
+    ) -> Result<()> {
+        self.validate()?;
+
+        let mut escrow = vesting_escrow.load_init()?;
+        escrow.init(
+            self.vesting_start_time,
+            self.cliff_time,
+            self.frequency,
+            self.cliff_unlock_amount,
+            self.amount_per_period,
+            self.number_of_period,
+            recipient,
+            mint,
+            sender,
+            base,
+            escrow_bump,
+            self.update_recipient_mode,
+            self.cancel_mode,
+            token_program_flag as u8,
```

Comment:
could you use token_program_flag.into()?

Reply:
updated